modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/SqlQueryProcessor.java

```diff
@@ -293,9 +294,11 @@ public synchronized CompletableFuture<Void> startAsync(ComponentContext componen
         var storageProfileValidator = new ClusterWideStorageProfileValidator(logicalTopologyService);
         var nodeFilterValidator = new ClusterWideNodeFilterValidator(logicalTopologyService);
 
+        CreateTableDefaultsView tablePropertiesView = clusterCfg.createTable().value();
```

Comment:
now configuration is cached once during component start, update won't be visible. We move this _inside_ supplier's body